packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/WorkflowSelectorContainer.spec.js

```diff
@@ -0,0 +1,27 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import { WorkflowSelectorContainer } from './WorkflowSelectorContainer'
+import WorkflowSelector from './WorkflowSelector'
+
+describe('Component > Hero > WorkflowSelector > WorkflowSelectorContainer', function () {
+  let wrapper
+  let componentWrapper
+
+  before(function () {
+    wrapper = shallow(<WorkflowSelectorContainer workflowDescription={WORKFLOW_DESCRIPTION} />)
```

Comment:
`WORKFLOW_DESCRIPTION` is undefined here.

Reply:
woops, did a cut instead of copy